deel-3-webpagina/index.html

```diff
@@ -7,10 +7,48 @@
     <title>Flexbox opdracht 3</title>
 </head>
 <body>
-    <div>
-        <h1>All Your Business Data in One Place</h1>
-        <h3>Sync and store all your business data. Explore your data with SQL or visualize it with your favorite BI or
-            analytics tools.</h3>
-    </div>
+<nav class="navbar">
+    <p>ANALYTICA</p>
+    <ul class="nav-links">
+        <li class="nav-item">About us</li>
+        <li class="nav-item">Strategy</li>
+        <li class="nav-item">Prices</li>
+        <li class="nav-item">Contact</li>
+    </ul>
+</nav>
+
+
+<div class="title-area">
+    <h1>All Your Business Data in One Place</h1>
+    <h3>Sync and store all your business data. Explore your data with SQL or visualize it with your favorite BI
+        or
+        analytics tools.</h3>
+</div>
+
+
```

Comment:
hier is nodig nog een out-container voor alle 
div elements 
-/html:
section class="content"
-/css:

.content {
display:  flex;
min-height:  60vh;
justify-content:  center;